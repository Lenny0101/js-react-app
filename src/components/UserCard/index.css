.userCard {
  display: flex;
  width: 100%;
  height: 60px;
  align-items: center;
  justify-content: flex-start;
}
.userImage img {
  width: 50px;
  height: 50px;
  border-radius: 50%;
  margin-right: 10px;
}
.userInfo p {
  font-color: #262626;
  margin-bottom: 0px;
  padding-bottom: 0px;
}
