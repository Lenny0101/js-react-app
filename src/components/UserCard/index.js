import React from "react";
import "./index.css";

const UserCard = ({ image, userName, userTime }) => (
  <div className="userCard">
    <div className="userImage">
      <img src={image} alt="" />
    </div>
    <div className="userInfo">
      <div className="userName">
        <p>{userName}</p>
      </div>
      <div className="userTime">
        <p>{userTime}</p>
      </div>
    </div>
  </div>
);

export default UserCard;
