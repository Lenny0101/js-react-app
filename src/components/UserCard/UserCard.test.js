import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import UserCard, { sum } from "./index";

describe("testing PostCard", () => {
  const wrapper = shallow(<UserCard />);

  //   it("sums two numbers", () => {
  //     expect(sum(3, 5)).toEqual(8);
  //   });
  //   it("renders properly", () => {
  //     expect(wrapper.length).toEqual(1);
  //   });
  it("renders a snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
