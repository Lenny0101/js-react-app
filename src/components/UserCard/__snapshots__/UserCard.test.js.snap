// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing PostCard renders a snapshot 1`] = `
<div
  className="userCard"
>
  <div
    className="userImage"
  >
    <img
      alt=""
    />
  </div>
  <div
    className="userInfo"
  >
    <div
      className="userName"
    >
      <p />
    </div>
    <div
      className="userTime"
    >
      <p />
    </div>
  </div>
</div>
`;
